blazor/diagram/swimlane/swimlane.md

```diff
@@ -140,6 +140,76 @@ The following code example explains how to customize the swimlane header.
 
 ![Swimlane Header Customization](Swimlane-images/Swimlane_Header_Customization.PNG)
 
+Swimlane header annotaiton also supports template. you can define a HTML content at tag level and specify the use of template with [UseTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.Annotation.html#Syncfusion_Blazor_Diagram_Annotation_UseTemplate) property. If you want to define separate template for each Swimlane, differentiate the annotaiton by the ID property.
+
+The following code example explains how to define a Swimlane header annotaiton template.
```

Comment:
The following code example explains how to define a Swimlane header annotation template.